proto/ssz_proto_library.bzl

```diff
@@ -47,7 +47,7 @@ minimal = {
     "extra_data.size": "32",
     "max_blobs_per_block.size": "6",
     "max_blob_commitments.size": "16",
-    "kzg_commitment_inclusion_proof_depth.size": "9",
+    "kzg_commitment_inclusion_proof_depth.size": "17",
```

Comment:
Please explain this change. Was there an upstream spec change for this?

Reply:
I believe it's this one: https://github.com/ethereum/consensus-specs/pull/3255

Reply:
This change fails test `//testing/spectest/minimal/deneb/ssz_static:go_default_test`

Reply:
the blob size will not change the proof size, the proof is for the KZG commitment, the block is smaller so it needs a smaller Merkle Tree, I'm pushing a revert of this and approving this PR